json/src/spec/authority_round.rs

```diff
@@ -58,6 +58,8 @@ pub struct AuthorityRoundParams {
 	pub maximum_empty_steps: Option<Uint>,
 	/// Strict validation of empty steps transition block.
 	pub strict_empty_steps_transition: Option<Uint>,
+	/// First block for which a 2/3 quorum (instead of 1/2) is required.
+	pub quorum_2_3_transition: Option<Uint>,
```

Comment:
Note that it will be called `quorum23Transition` in a json spec, which looks a bit weird.
What do you think about `two_thirds_majority_transition`?

Reply:
Done.